src/index.js

```diff
@@ -76,19 +76,16 @@ function serve (options = { contentBase: '' }) {
     server = createServer(requestListener).listen(options.port, options.host)
   }
 
-  let running = options.verbose === false
-
   return {
     name: 'serve',
     generateBundle () {
-      if (!running) {
-        running = true
-
         // Log which url to visit
         const url = (options.https ? 'https' : 'http') + '://' + (options.host || 'localhost') + ':' + options.port
-        options.contentBase.forEach(base => {
-          console.log(green(url) + ' -> ' + resolve(base))
-        })
+        if (options.verbose) {
```

Comment:
Verbose should be enabled by default
```suggestion
        if (options.verbose !== false) {
```

Reply:
Fixed!

Reply:
Something went wrong here 😛  
I tweaked it a little, here is the end result: https://github.com/thgh/rollup-plugin-serve/commit/f6474463bbeb938a13457f8ffafe5d59121e7c0e